patch/arm64-sonic-kernel-config.patch

```diff
@@ -0,0 +1,34 @@
+From 47b5ed9669102ef6b94be47df9cd5ddeb03e47b6 Mon Sep 17 00:00:00 2001
+From: Antony Rheneus <arheneus@marvell.com>
+Date: Wed, 25 Sep 2019 23:58:54 +0530
+Subject: [PATCH] arm64 sonic kernel config
+
+---
+ .config | 14 +++++++++++++
+ 1 file changed, 14 insertions(+)
+
+diff --git a/debian/build/build_arm64_none_arm64/.config b/debian/build/build_arm64_none_arm64/.config
+index 0ce4748..c5e6562 100644
+--- a/debian/build/build_arm64_none_arm64/.config
++++ b/debian/build/build_arm64_none_arm64/.config
+@@ -7265,3 +7265,17 @@ CONFIG_FONT_AUTOSELECT=y
+ CONFIG_SG_POOL=y
+ CONFIG_ARCH_HAS_SG_CHAIN=y
```

Comment:
why cannot we use the same config for both arm64 and amd64?

Reply:
We cannot use as arch differs in kernel config.
It is maintained in the debian package https://sonicstorage.blob.core.windows.net/packages/kernel-source/linux_4.9.168-1+deb9u5.dsc?sv=2015-04-05&sr=b&sig=ZcTpvRltWLWwnJn3vQ%2BgTP1dVF6QSinOCJ1FSuyiogU%3D&se=2033-04-28T06%3A14%3A30Z&sp=r

Refer the targets mentioned in 
linux-4.9.168/debian/rules.gen